File: 14_CursoProgramOrientObjects/Java/Main.java


class Main {
    public static void main(String[] args) {
        System.out.println("Hola Mundo");
        System.out.println("Car...");
        Car car = new Caaar("AMQ123", new Account("Andres Herrera", "ADN1234", "andresH@hotmail.com", "12365"));
        car.passenger = 4;
        car.printDataCar();

        System.out.println("UberX...");
        UberX uberx = new UberX("MKL185", new Account("Maria Loyola", "AJKL12334", "mariaL@hotmail.com", "98766"), "Toyota", "Corolla");
        
    }
}

File: 14_CursoProgramOrientObjects/Python/main.py
from car import Car
from account import Account
from uberX import UberX
from user import User

if __name__ == "__main__":
    print("Inicializando las info de los carros")
    print("Car")

    car = Car("AMS234", Account("Andres Herrera", "AND876", "andres@platzi.com", "2563"))
    print(vars(car))
    print(vars(car.driver))

    print("UberX")
    uberX = UberX("KLO365", Account("Marco Diaz", "SGHJ1234", "marco@platzi.com","7856"), "Toyota", "Corolla")
    print(vars(uberX))
    print(vars(uberX.driver))

    print("User")
    user = User("Mariana Diaz", "SDF1234", "mariana@platzi.com", "7856")
    print(vars(user))